CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(qskinny VERSION 999.99.9) # TODO get from git branch name or 
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+# force include Qt's private headers
+set(Qt5Core_FIND_COMPONENTS          Private CACHE STRING "force include private headers" FORCE)
+set(Qt5Gui_FIND_COMPONENTS           Private CACHE STRING "force include private headers" FORCE)
+set(Qt5OpenGL_FIND_COMPONENTS        Private CACHE STRING "force include private headers" FORCE)
+set(Qt5Quick_FIND_COMPONENTS         Private CACHE STRING "force include private headers" FORCE)
+set(Qt5QuickWidgets_FIND_COMPONENTS  Private CACHE STRING "force include private headers" FORCE)
+set(Qt5Svg_FIND_COMPONENTS           Private CACHE STRING "force include private headers" FORCE)
+set(Qt5WebEngine_FIND_COMPONENTS     Private CACHE STRING "force include private headers" FORCE)
+set(Qt5WebEngineCore_FIND_COMPONENTS Private CACHE STRING "force include private headers" FORCE)
+set(Qt5Widgets_FIND_COMPONENTS       Private CACHE STRING "force include private headers" FORCE)
+
```

Comment:
Check and minimize private header / target inclusion

- G:\repositories\qskinny\inputcontext\CMakeLists.txt -> Qt5::GuiPrivate
- G:\repositories\qskinny\playground\webview\CMakeLists.txt -> Qt5::WebEnginePrivate
- G:\repositories\qskinny\src\CMakeLists.txt -> Qt5::CorePrivate 
- G:\repositories\qskinny\src\CMakeLists.txt -> Qt5::QuickPrivate
- G:\repositories\qskinny\tools\svg2qvg\CMakeLists.txt -> Qt5::GuiPrivate